### ISSUE TITLE
Change #1104 included in 0.14.2 release notes but was actually in 0.14.3

### ISSUE THREAD

**Issue by timw6n** [holoviz/datashader#1376]
This is the change to the dask dependencies. 

Just a very minor issue in the release notes at https://datashader.org/releases.html rather than the code itself.

Thought I'd raise this as caused a bit of confusion this morning when working through our application's dependency tree.

Looks like #1104 actually landed for 0.14.3 rather than 0.14.2  - see https://github.com/holoviz/datashader/compare/v0.14.2...v0.14.3#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7

**Comment by hoxbro** [comment]
Are you sure? As far as I can see #1104 did land in 0.14.2.

**Comment by timw6n** [comment]
I've just pulled the tarballs from PyPI to double check and it definitely looks like 0.14.2 is still depending on dask[complete] when 0.14.3 moved to dask.

```
twaterson@twaterson-XPS-9520:~/Downloads$ cat datashader-0.14.2/setup.py | grep dask | head -n 1
    'dask[complete]',
twaterson@twaterson-XPS-9520:~/Downloads$ cat datashader-0.14.3/setup.py | grep dask | head -n 1
    'dask',
```

But I see what happened is that #1104 was in 0.14.2 but then #1107 (also 0.14.2) effectively reverted it. That was then fixed in 0.14.3 by #1111. So you're right the release notes have the correct sequence of PRs.

All good.
